`TestResizeTerminal` flakiness

## Failure
Saw this fail off of `branch/v10`.

### CI Logs
- https://console.cloud.google.com/cloud-build/builds/2fd03e82-d9fa-45e7-b5b9-a67817d56c93;step=0?project=ci-account
- https://console.cloud.google.com/cloud-build/builds/c1df481f-cbc6-458f-b55c-4e054e9edfc3?project=ci-account
- https://github.com/gravitational/teleport/actions/runs/9226959697/job/25387872605 

### Relevant Snippet
```
FAIL: github.com/gravitational/teleport/lib/web.TestResizeTerminal
===================================================
OUTPUT github.com/gravitational/teleport/lib/web.TestResizeTerminal
===================================================
=== RUN   TestResizeTerminal
=== PAUSE TestResizeTerminal
=== CONT  TestResizeTerminal
=== CONT  TestResizeTerminal
    apiserver_test.go:1057: 
        	Error Trace:	apiserver_test.go:1057
        	Error:      	Received unexpected error:
        	            	timeout waiting for raw event
        	Test:       	TestResizeTerminal
```

Another hit but the reason is different:
```
=== CONT  TestResizeTerminal
    testing.go:1097: TempDir RemoveAll cleanup: unlinkat /tmp/TestResizeTerminal3559055086/002/log/upload/streaming/default/multi/713ac636-7d24-46d5-a80e-ada49172ab5e/d2e590b1-1823-424d-aad4-12c5c1018255: directory not empty
--- FAIL: TestResizeTerminal (8.59s)
```
https://console.cloud.google.com/cloud-build/builds/823f60bd-8f87-4dd4-8651-89111a266fdf?project=ci-account

```
=== CONT  TestResizeTerminal
{"caller":"srv/sess.go:926","component":"session:node","level":"debug","message":"Copying from PTY to writer completed with error read /dev/ptmx: input/output error.","session_id":"972cf399-a3a1-418c-8c45-864c6c3bea70","timestamp":"2023-01-27T08:03:23Z"}
    testing.go:1097: TempDir RemoveAll cleanup: unlinkat /tmp/TestResizeTerminal3347799852/002/log/upload/streaming/default/multi/d84f701a-47e0-4e48-8729-ac039c0e6238/972cf399-a3a1-418c-8c45-864c6c3bea70: directory not empty
--- FAIL: TestResizeTerminal (6.71s)
```

https://github.com/gravitational/teleport/actions/runs/4022563145/jobs/6912431643

--- a/lib/web/apiserver_test.go
+++ b/lib/web/apiserver_test.go
@@ -1229,7 +1229,7 @@ func TestNewTerminalHandler(t *testing.T) {
 
 func TestResizeTerminal(t *testing.T) {
 	t.Parallel()
-	s := newWebSuite(t)
+	s := newWebSuiteWithConfig(t, webSuiteConfig{disableDiskBasedRecording: true})
 	sid := session.NewID()
 
 	errs := make(chan error, 2)

